### ISSUE TITLE
Game crashed when I input text in filter field in "Choose Map" dialog

### ISSUE THREAD

**Issue by Somewater** [OpenRA/OpenRA#9698]
After installation of some map game crashed after start of filtering maps by name, using input filed.

Exceptional log:
```
Red Alert Mod at Version release-20150919
on map 04936ef1e2b49e43291c3c42d13bcaa2e089c00f (Desert Shellmap by Scott_NZ).
Operating System: Linux (Unix 3.13.0.58)
Runtime Version: Mono 3.2.8 (Debian 3.2.8+dfsg-4ubuntu1.1) CLR 4.0.30319.17020
Exception of type `System.NullReferenceException`: Object reference not set to an instance of an object
  at OpenRA.Mods.Common.Widgets.Logic.MapChooserLogic+<EnumerateMaps>c__AnonStorey7.<>m__1 (OpenRA.MapPreview m) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateWhereIterator>c__Iterator1E`1[OpenRA.MapPreview].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable.ToArray[MapPreview] (IEnumerable`1 source) [0x00000] in <filename unknown>:0 
  at System.Linq.QuickSort`1[OpenRA.MapPreview]..ctor (IEnumerable`1 source, System.Linq.SortContext`1 context) [0x00000] in <filename unknown>:0 
  at System.Linq.QuickSort`1+<Sort>c__Iterator0[OpenRA.MapPreview].MoveNext () [0x00000] in <filename unknown>:0 
  at OpenRA.Mods.Common.Widgets.Logic.MapChooserLogic.EnumerateMaps (MapClassification tab, OpenRA.Mods.Common.Widgets.ScrollItemWidget template) [0x00000] in <filename unknown>:0 
  at OpenRA.Mods.Common.Widgets.Logic.MapChooserLogic+<MapChooserLogic>c__AnonStorey0.<>m__4 () [0x00000] in <filename unknown>:0 
  at OpenRA.Mods.Common.Widgets.TextFieldWidget.HandleTextInput (System.String text) [0x00000] in <filename unknown>:0 
  at OpenRA.Widgets.Widget.HandleTextInputOuter (System.String text) [0x00000] in <filename unknown>:0 
  at OpenRA.Widgets.Ui.HandleTextInput (System.String text) [0x00000] in <filename unknown>:0 
  at OpenRA.DefaultInputHandler+<OnTextInput>c__AnonStorey1.<>m__0 () [0x00000] in <filename unknown>:0 
  at OpenRA.Sync.CheckSyncUnchanged[Boolean] (OpenRA.World world, System.Func`1 fn) [0x00000] in <filename unknown>:0 
  at OpenRA.DefaultInputHandler.OnTextInput (System.String text) [0x00000] in <filename unknown>:0 
  at OpenRA.Renderer.Sdl2.Sdl2Input.PumpInput (IInputHandler inputHandler) [0x00000] in <filename unknown>:0 
  at OpenRA.Renderer.Sdl2.Sdl2GraphicsDevice.PumpInput (IInputHandler inputHandler) [0x00000] in <filename unknown>:0 
  at OpenRA.Graphics.Renderer.EndFrame (IInputHandler inputHandler) [0x00000] in <filename unknown>:0 
  at OpenRA.Game.RenderTick () [0x00000] in <filename unknown>:0 
  at OpenRA.Game.Loop () [0x00000] in <filename unknown>:0 
  at OpenRA.Game.Run () [0x00000] in <filename unknown>:0 
  at OpenRA.Program.Run (System.String[] args) [0x00000] in <filename unknown>:0 
  at OpenRA.Program.Main (System.String[] args) [0x00000] in <filename unknown>:0 
```

**Comment by obrakmann** [comment]
Thanks for the report. This has already been fixed in #9425. Until that fix is released, try removing [this map](http://resource.openra.net/maps/5923/) from your openra map directory.

**Comment by Somewater** [comment]
Thanks!
